src/main/java/ch/njol/skript/effects/EffChange.java

```diff
@@ -27,71 +20,77 @@
-		"add argument to {blacklist::*}",
-		"give a diamond pickaxe of efficiency 5 to the player",
-		"increase the data value of the clicked block by 1",
-		"# remove:",
-		"remove 2 pickaxes from the victim",
-		"subtract 2.5 from {points::%uuid of player%}",
-		"# remove all:",
-		"remove every iron tool from the player",
-		"remove all minecarts from {entitylist::*}",
-		"# delete:",
-		"delete the block below the player",
-		"clear drops",
-		"delete {variable}",
-		"# reset:",
-		"reset walk speed of player",
-		"reset chunk at the targeted block"})
-@Since("1.0 (set, add, remove, delete), 2.0 (remove all)")
+@Description({
+	"A very general effect that can change many <a href='./expressions'>expressions</a>.",
+	"Many expressions can only be set and/or deleted, while some can have things added to or removed from them."
+})
+@Examples({
+	"# set:",
+	"Set the player's display name to \"&lt;red&gt;%name of player%\"",
+	"set the block above the victim to lava",
+	"# add:",
+	"add 2 to the player's health # preferably use '<a href='#EffHealth'>heal</a>' for this",
+	"add argument to {blacklist::*}",
+	"give a diamond pickaxe of efficiency 5 to the player",
+	"increase the data value of the clicked block by 1",
+	"# remove:",
+	"remove 2 pickaxes from the victim",
+	"subtract 2.5 from {points::%uuid of player%}",
+	"# remove all:",
+	"remove every iron tool from the player",
+	"remove all minecarts from {entitylist::*}",
+	"# delete:",
+	"delete the block below the player",
+	"clear drops",
+	"delete {variable}",
+	"# reset:",
+	"reset walk speed of player",
+	"reset chunk at the targeted block"
+})
+@Since({
+	"1.0 (set, add, remove, delete)",
+	"2.0 (remove all)"
+})
 public class EffChange extends Effect {
-	private static Patterns<ChangeMode> patterns = new Patterns<>(new Object[][] {
-			{"(add|give) %objects% to %~objects%", ChangeMode.ADD},
-			{"increase %~objects% by %objects%", ChangeMode.ADD},
-			{"give %~objects% %objects%", ChangeMode.ADD},
+	private static final Patterns<ChangeMode> patterns = new Patterns<>(new Object[][] {
+		{"(add|give) %objects% to %~objects%", ChangeMode.ADD},
+		{"increase %~objects% by %objects%", ChangeMode.ADD},
+		{"give %~objects% %objects%", ChangeMode.ADD},
 
-			{"set %~objects% to %objects%", ChangeMode.SET},
+		{"set %~objects% to %objects%", ChangeMode.SET},
 
-			{"remove (all|every) %objects% from %~objects%", ChangeMode.REMOVE_ALL},
+		{"remove (all|every) %objects% from %~objects%", ChangeMode.REMOVE_ALL},
 
-			{"(remove|subtract) %objects% from %~objects%", ChangeMode.REMOVE},
-			{"(reduce|decrease) %~objects% by %objects%", ChangeMode.REMOVE},
+		{"(remove|subtract) %objects% from %~objects%", ChangeMode.REMOVE},
+		{"(reduce|decrease) %~objects% by %objects%", ChangeMode.REMOVE},
 
-			{"(delete|clear) %~objects%", ChangeMode.DELETE},
+		{"(delete|clear) %~objects%", ChangeMode.DELETE},
 
-			{"reset %~objects%", ChangeMode.RESET}
+		{"reset %~objects%", ChangeMode.RESET}
```

Comment:
any statement spread over multiple lines needs to have the extra lines indented twice by code convention